### PR TITLE
Reconfig ESlint for dashboard sub-project

### PR DESCRIPTION
**Pull request by JiaHua-Zou**
## Issue This PR Addresses
#2999 

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description
The `ESlint` setting for dashboard has been move to it's sub-project.

## Steps to test the PR
1. 'pnpm install`
2. `pnpm lint`

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
image

- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by JiaHua-Zou** [comment]
`E2E` test fail. Need to go back and fix the `package.json` file. The dependency `@senecacdot/eslint-config-telescope` is wrong.
image

**Comment by cindyledev** [comment]
Wasn't @aserputov assigned this issue? How come you're submitting a PR, @JiaHua-Zou? Not that I mind, I hate all the ESLint warnings, just curious.

**Comment by aserputov** [comment]
@JiaHua-Zou we have one issue for both of us and the rest are only assigned for me.

**Comment by JiaHua-Zou** [comment]
Yea it was assign for both of us. I guess in the meeting we forgot to put me there. @cindyledev

**Comment by cindyledev** [comment]
Ah okay, I'm glad to see these ESLint issues getting addressed. They've been driving me bonkers

File: src/api/status/.eslintrc.js [hunk 1]
```diff
@@ -1,6 +1,15 @@
 module.exports = {
-  env: {
-    browser: true,
-    node: true,
-  },
+  extends: '@senecadot/eslint-config-telescope',
```
Comment (cindyledev) [inline src/api/status/.eslintrc.js:2]: `extends: '@senecacdot/eslint-config-telescope',`

missing a `c`

**Comment by cindyledev** [comment]
Once you fix the minor typo, I'll approve. This PR gets rid of the ESLint errors in `/src/api/status/*` for me :)

File: src/api/status/.eslintrc.js (hunk 1 above)
Comment (tpmai22) [reply src/api/status/.eslintrc.js:2]: I just curious when looking into this PR. It work even with a typo ?

Comment (JiaHua-Zou) [reply src/api/status/.eslintrc.js]: I think it knows because the in the `pnpm-lock.yaml` file has the path link to the dependency.

Comment (tpmai22) [reply src/api/status/.eslintrc.js]: @JiaHua-Zou Awesome. Good to know !

**Review by cindyledev — APPROVED** [review]

**Review by tpmai22 — APPROVED** [review]
Thank you for jumping on this ! LGTM !

**Pull request merged**